lib/python/treadmill_aws/api/host.py

```diff
@@ -35,7 +35,8 @@ def create_hosts(ami,
                 secgroup_ids=secgroup,
                 instance_type=size,
                 subnet_id=subnet,
-                disk=disk)
+                disk=disk,
+                instance_vars={})
```

Comment:
instance_vars=None)

if not instance_vars:
  instance_vars = {}

Reply:
OK, changed to None.